account/source/main.c

```diff
@@ -37,7 +37,21 @@ int main(int argc, char **argv)
         rc = accountGetPreselectedUser(&userID);
 
         if (R_FAILED(rc)) {
-            printf("accountGetPreselectedUser() failed: 0x%x\n", rc);
+            printf("accountGetPreselectedUser() failed: 0x%x, using pselUi..\n", rc);
+            
+            /* Create player selection UI settings */
+            PselUiSettings settings;
```

Comment:
Can just use pselShowUserSelector instead.

Reply:
Ah, TIL, will look into it.